src/components/MnemonicInput/MnemonicInputTs.ts

```diff
@@ -0,0 +1,118 @@
+import { UIHelpers } from '@/core/utils/UIHelpers.ts';
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
+// internal dependencies
+
+@Component
+export class MnemonicInputTs extends Vue {
+  /**
+   * @description:bind the input
+   */
+  public inputWord: string = '';
+
+  /**
+   * @description: wordsArray
+   */
+  public wordsArray = [];
+
+  /**
+   * @description: status of isEditing 
+   */
+  public isEditing: boolean = false;
+
+  /**
+   * @description: isNeedPressDelTwice
+   */
+  public isNeedPressDelTwice = true;
+
+  /**
+   * @description: watch the inputform
+   */
+  @Watch('inputWord')
+  watchFormItems(newVal: string, oldVal: string) {
+    //add the limit
+    if (this.wordsArray.length >= 24) {
+      this.inputWord = '';
+      this.initInput();
+    } else {
+      //control the keyboard input rules
+      this.inputWord = newVal.replace(/[^a-zA-Z]/g, '')
+      //determine if the input is editing status
+      if (!this.isEditing && !!this.inputWord) {
+        this.isEditing = true
+      }
+    }
```

Comment:
instead of a **watch**, it is preferred to use computed properties.

```
public  get userInput(): string {
    return this.inputWord || ''
}

public set userInput(input: string) {
    // do modifications/split here 
}
```